packages/block-library/src/cover/edit.js

```diff
@@ -69,11 +73,69 @@ function retrieveFastAverageColor() {
 	return retrieveFastAverageColor.fastAverageColor;
 }
 
+const RESIZABLE_BOX_ENABLE_OPTION = {
+	top: false,
+	right: false,
+	bottom: true,
+	left: false,
+	topRight: false,
+	bottomRight: false,
+	bottomLeft: false,
+	topLeft: false,
+};
+
+function ResizableCover( {
+	className,
+	children,
+	onResize,
+	onResizeStop,
+} ) {
+	const [ isResizing, setIsResizing ] = useState( false );
+	const onResizeEvent = useCallback(
+		( event, direction, elt ) => {
+			onResize( elt.clientHeight );
+		},
+		[ onResize ],
+	);
+	const onResizeStartEvent = useCallback(
+		() => {
+			setIsResizing( true );
```

Comment:
there is a weird jump if the content is higher then the current height and we resize to below it, the cover will indeed expand again to fit the content, but if you click the resize handle, there will be a jump

Reply:
Nice catch problem addressed 👍